galite-tests/src/test/kotlin/org/kopi/galite/demo/client/ClientForm.kt

```diff
@@ -27,8 +27,8 @@ import org.kopi.galite.report.Report
 
 object ClientForm : ReportSelectionForm() {
   override val locale = Locale.FRANCE
-  override val title = "client form"
-  val page = page("page")
+  override val title = "Clients"
```

Comment:
This class is to be fixed in my store application open PR, not here. (already fixed you can review it and merge into master)